src/Apps/Artwork/Components/ArtworkDetails/RequestConditionReport.tsx

```diff
@@ -0,0 +1,205 @@
+import { Button, Col, Modal, Row, Sans, Serif, Spinner } from "@artsy/palette"
+import React, { useState } from "react"
+import { commitMutation, graphql, RelayProp } from "react-relay"
+
+import { AnalyticsSchema, track, useSystemContext, useTracking } from "Artsy"
+import * as Schema from "Artsy/Analytics/Schema"
+import { ErrorModal } from "Components/Modal/ErrorModal"
+import * as authentication from "Components/NavBar/Utils/authentication"
+import createLogger from "Utils/logger"
+
+import { RequestConditionReportMutation } from "__generated__/RequestConditionReportMutation.graphql"
+
+const logger = createLogger(
+  "Apps/Artwork/Components/ArtworkDetails/RequestConditionReport"
+)
+
+export interface RequestConditionReportProps {
+  saleArtworkID: string
+  relay?: RelayProp
+}
+
+export const RequestConditionReport: React.FC<RequestConditionReportProps> = track(
+  {
+    context_module: AnalyticsSchema.ContextModule.RequestConditionReport,
+  }
+)(props => {
+  const { mediator, user } = useSystemContext()
+  const { trackEvent } = useTracking()
+
+  const [requestingConditionReport, setRequestingConditionReport] = useState(
+    false
+  )
+
+  const [
+    showConditionReportRequestedModal,
+    setShowConditionReportRequestedModal,
+  ] = useState(false)
+
+  const [showErrorModal, setShowErrorModal] = useState(false)
+
+  const isLoggedIn = Boolean(user)
+  const { saleArtworkID, relay } = props
+
+  const requestConditionReport = () => {
+    return new Promise(async (resolve, reject) => {
+      commitMutation<RequestConditionReportMutation>(relay.environment, {
+        onCompleted: data => {
+          resolve(data)
+        },
+        onError: error => {
+          reject(error)
+        },
+        mutation: graphql`
+          mutation RequestConditionReportMutation(
+            $input: RequestConditionReportInput!
+          ) {
+            requestConditionReport(input: $input) {
+              conditionReportRequest {
+                internalID
+              }
+            }
+          }
+        `,
+        variables: {
+          input: { saleArtworkID },
+        },
+      })
+    })
+  }
+
+  const handleMutationError = error => {
+    logger.error(error)
+
+    let errorMessages: string[]
+    if (Array.isArray(error)) {
+      errorMessages = error.map(e => e.message)
+    } else if (typeof error === "string") {
+      errorMessages = [error]
+    } else if (error.message) {
+      errorMessages = [error.message]
+    }
+
+    setRequestingConditionReport(false)
+    trackRequestConditionReportFailure(errorMessages)
+    setShowErrorModal(true)
+  }
+
+  const commonProperties = {
```

Comment:
Is it possible to promote any of these properties to be arguments to [`track()`](https://github.com/artsy/reaction/pull/3099/files#diff-578cbd16d0663ec14ecf363c0f808a66R23-R25)?

Reply:
Ahhh, likely. I'll give that a try.

Reply:
I believe the ConfirmBid component is the latest understanding how we can track common properties. https://github.com/artsy/reaction/blob/master/src/Apps/Auction/Routes/ConfirmBid/index.tsx#L361-L367

Here's a commit that should have some helpful context here: 926455ac268e13e8661e2dee5905169064016d17